oshdb-util/src/main/java/org/heigit/bigspatialdata/oshdb/util/geometry/OSHDBGeometryBuilder.java

```diff
@@ -279,24 +279,31 @@ private static Geometry getMultiPolygonGeometry(
       T entity, OSHDBTimestamp timestamp, TagInterpreter areaDecider, P clipPoly) {
     Geometry geom = OSHDBGeometryBuilder.getGeometry(entity, timestamp, areaDecider);
     return Geo.clip(geom, clipPoly);
+  } 
+  
+  /**
+   * Create the Geometry of an OSHDBBoundingBox. Will always contain 5 Coordinates, even for
+   * point-like bbx -> point-like polygons for bbx of a Point.
+   *
+   * @param bbox The BoundingBox the polygon should be created for.
+   * @return com.vividsolutions.jts.geom.Geometry Returns JTS geometry object for convenience.
+   * Result is an empty polygon for a null BBX.
+   */
+  public static Polygon getGeometry(OSHDBBoundingBox bbox) {
+    GeometryFactory gf = new GeometryFactory();
+    if (bbox == null) {
```

Comment:
why the extra check against `null` here? As far as I know, we never generate null bounding boxes.
Maybe add an assert instead (`assert bbox != null : "a bounding box is not allowed to be null";` or so)?

Reply:
yes I thought of that too. Will implement...